scss/civicrm/contact/_tabs.scss

```diff
@@ -1,194 +1,195 @@
-/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+/* stylelint-disable selector-max-id */
```

Comment:
Is this against CODING.md rule when we say that it is a must to have all rules disabled? I may understand why you do this, but should we then say in CODING.md that `if you *can* enable some rules, please go ahead`?

Reply:
yes i actually regret adding that get-out-of-jail-free line in CODING.md. Of course instead it should say that it's ok to disable some rules, _if necessary_. There are plans to overhaul the documentation in the readme, i'll fix that line then (i need to come up with a good wording and maybe some examples, etc)